src/cli/config.ts

```diff
@@ -26,6 +35,18 @@ export interface Plugin {
   addConfigImport: boolean;
 }
 
+
+/**
+ * The '@nuxtjs/tailwindcss' module helps you set up Tailwind CSS (version 3) in your Nuxt 3 application in seconds.
+ */
+export const supportedNuxtModule: readonly Module[] = [
```

Comment:
This NuxtTailwind module seems to work out of the box, let's not ask users about it.

They select Nuxt with this Module.

So please, let's remove this and all related code and simply add the dependency into the Config.

Reply:
Okay, I will remove it ✨